############### Distance estimation parameters ###############
object_radius_known: true
object_radius: 0.25
# distance estimation from object size parameters
max_dist: 30.0
max_dist_diff: 4.0
# depthmap parameters
min_depth: 200 # millimetres
max_depth: 12000 # millimetres
# xy_covariance_coeff: 0.001
# z_covariance_coeff: 0.005

############### Default parameters for the dynamic reconfigure ################
filter_by_area: true
filter_by_circularity: false
filter_by_convexity: false
filter_by_inertia: true
max_area: 150000.0
max_circularity: 1.0
max_convexity: 1.0
max_inertia_ratio: 1.0
min_area: 20.0
min_circularity: 0.4
min_convexity: 0.0
min_dist_between: 10.0
min_inertia_ratio: 0.2
min_repeatability: 1
threshold_step: 255.0
detector: 0
dilate_iterations: 0
fill_holes: 0
filter_by_color: true
gaussianblur_size: 5
hough1: 500
hough2: 15.0
id: 1
medianblur_size: 0

## SIMULATION BALLOONS
# # RED objectS
# color_center: 179
# color_range: 15
# min_brightness: 1
# min_saturation: 100

# # GREEN BALLOONS
# color_center: 61
# color_range: 15
# min_brightness: 1
# min_saturation: 100

# # ORANGE BALLOONS
# color_center: 20
# color_range: 15
# min_brightness: 1
# min_saturation: 100
